backend/src/LibCloud/SqlCompiler.fs

```diff
@@ -727,6 +734,7 @@ let rec lambdaToSql
             match t with
             | TString -> "text"
             | TInt -> "bigint"
+            | TInt8 -> "bigint"
```

Comment:
Using a `tinyint` and `| TInt8, DInt8 i -> return Sql.int8  i, TInt8`
instead of `bigint` and `| TInt8, DInt8 i -> return Sql.int64 (int i), TInt8`
results in the following error
```
The CLR type System.SByte isn't natively supported by Npgsql or your PostgreSQL. To use it with a PostgreSQL composite you need to specify DataTypeName or to map it, please refer to the documentation
```

Reply:
I don't think postgres has a tinyint type, which might cause this.

Reply:
should we use the closest which is a smallint?